src/main/resources/api/sponsor.json

```diff
@@ -0,0 +1,36 @@
+{
+  "meta": {
+    "count": 1
+  },
+  "data": [
```

Comment:
Please add one more resource(sponsor) so that we can confirm if array is working correctly or not in future because actually server response will contain array.